src/js/editor/index.js

```diff
@@ -0,0 +1,15 @@
+import toolbox from './toolbox/index';
+import singleEvents from './single_item/index';
+import * as todoEvents from './todo_events';
+
+const init = (itemId) => {
+  todoEvents.getAll().then((items) => {
+    toolbox(items, itemId);
+
+    const itemToShow = !itemId ? items[0] : items.find((item) => item.id === itemId);
+
+    singleEvents(itemToShow);
```

Comment:
czytając ten kod nie wiem co robi `singleEvents`

Reply:
Tutaj za to treść danej notatki